cmd/geth/main.go

```diff
@@ -417,6 +428,35 @@ func attach(ctx *cli.Context) {
 	}
 }
 
+func initGenesis(ctx *cli.Context) {
```

Comment:
Would be nice to have a method doc here, even if just copy pasting the usage string from the flag.